packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -3489,7 +3543,7 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
       text = widget.obscuringCharacter * text.length;
       // Reveal the latest character in an obscured field only on mobile.
```

Comment:
This comment has to be updated. Also a good place to explain why iOS is excluded, maybe add the issue link?

Reply:
Yeah that was for Android only.